readme.txt

```diff
@@ -0,0 +1,27 @@
+=== Wikipedia Preview ===
+Contributors: Wikimedia Foundation
+Donate link: 
+Tags: wikipedia, facts, popup, card, wiki
+Stable tag: trunk
+License: MIT
+License URI: https://github.com/wikimedia/wikipedia-preview/blob/main/LICENSE
+
+Wikipedia Preview allows you to show a popup card with a short summary from Wikipedia when a reader clicks or hovers over a link
+
+== Description ==
+Wikipedia Preview is a plugin that brings free knowledge to your site in the form of an Wikipedia article preview popup. Your readers will be able to simply click or hover over a word and directly see a short summary from the linked Wikipedia article. Wikipedia Preview is an official plugin developed and supported by the Wikimedia Foundation.  
+
+**Features**
+
+* Handles any link to a Wikipedia article regardless of language, lead image presence or length
+* Works for Left-To-Right (LTR) languages
```

Comment:
it works for RTL languages too

Reply:
You're right, I left that out because I thought it was implicit but now that you bring it up I don't see why not be explicit about that